src/components/BrowseBikesListItem.jsx

```diff
@@ -1,81 +1,123 @@
-    <Grid style={{ flex: 1 }}>
-      <Row>
-        <Col>
-          <TouchableOpacity
-            onPress={() => {
-              setSelectedBikeID(bike.id);
-            }}
-          >
-            <Thumbnail circle source={{ uri: bike.picUrl }} />
-          </TouchableOpacity>
-        </Col>
-        <Col>
-          <TouchableOpacity
-            onPress={() => {
-              setSelectedBikeID(bike.id);
-            }}
-          >
-            <Text>Frame</Text>
-            <Text note>{bike.frame}</Text>
-          </TouchableOpacity>
-        </Col>
-        <Col>
-          <TouchableOpacity
-            onPress={() => {
-              setSelectedBikeID(bike.id);
-            }}
-          >
-            <Text>Style</Text>
-            <Text note>{bike.style}</Text>
-          </TouchableOpacity>
-        </Col>
-        <Col>
-          <TouchableOpacity
-            onPress={() => {
-              setSelectedBikeID(bike.id);
-            }}
-          >
-            <Text>Distance</Text>
-            <Text note>
-              {haversine({ latitude: bike.latitude, longitude: bike.longitude }, location, {
+const BrowseBikesListItem = ({ bike, navigation, selectedBikeID, setSelectedBikeID, location }) => {
+  const [rating, setRating] = useState();
+  const [err, setErr] = useState();
+
+  useEffect(() => {
+    getBikeRatingCountAndAvg(bike.id)
+      .then((bikeRating) => {
+        setRating(bikeRating.score);
+      })
+      .catch((e) => setErr(e));
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  if (err) {
+    return (
+      <Container>
+        <Text>There was an unexpected error</Text>
+        <Text>{err}</Text>
+      </Container>
+    );
+  }
+
+  return (
+    <ListItem keyExtractor={{ item: bike, index: bike.id }} selected={bike.id === selectedBikeID}>
+      <Grid style={{ flex: 1 }}>
+        <Row>
+          <Col>
+            <TouchableOpacity
+              onPress={() => {
+                setSelectedBikeID(bike.id);
+              }}
+            >
+              <Thumbnail circle source={{ uri: bike.picUrl }} />
+            </TouchableOpacity>
+          </Col>
+          <Col>
+            <TouchableOpacity
+              onPress={() => {
+                setSelectedBikeID(bike.id);
+              }}
+            >
+              <Text>Frame</Text>
+              <Text note>{bike.frame}</Text>
+            </TouchableOpacity>
+          </Col>
+          <Col>
+            <TouchableOpacity
+              onPress={() => {
+                setSelectedBikeID(bike.id);
+              }}
+            >
+              <Text>Style</Text>
+              <Text note>{bike.style}</Text>
+            </TouchableOpacity>
+          </Col>
+          <Col>
+            <TouchableOpacity
+              onPress={() => {
+                setSelectedBikeID(bike.id);
+              }}
+            >
+              <Text>Dist</Text>
```

Comment:
We can get away with changing this back to distance now that the style column is gone.